app/assets/stylesheets/hyku.scss

```diff
@@ -19,29 +19,28 @@
 @media (min-width: $screen-sm-min) {
   .logo-set {
     display: flex;
+    flex-direction: column;
     justify-content: center;
     flex-direction: column;
   }
-
   // sets the home page tabs to use flex rather than columns
   // allows full width if tabs are turned off with feature flipper
   .home-content {
     display: flex;
     justify-content: space-between;
   }
-
   .home-tabs-left {
     min-width: 50%;
   }
 }
 
 #banner.jumbotron {
-  background: image_url('white-cloud-background.jpg') no-repeat;
-  background-color: $gray-lighter;
+  background: image_url("Hyku_Commons_Logo.png") no-repeat;
```

Comment:
This should be the white-cloud-background file and not the hyku commons logo

Reply:
done 👍